firefox-ios/Client/Frontend/TabContentsScripts/LoginsHelper.swift

```diff
@@ -338,6 +338,11 @@ class LoginsHelper: TabContentScript {
                                      object: .loginsAutofilled)
     }
 
+    public static func yieldFocusBackToField(with tab: Tab) {
+        let jsFocusCallback = "window.__firefox__.logins.yieldFocusBackToField()"
+        tab.webView?.evaluateJavascriptInDefaultContentWorld(jsFocusCallback)
+    }
+
```

Comment:
This is for [FXIOS-8690](https://mozilla-hub.atlassian.net/browse/FXIOS-8690). When we dismiss the bottom sheet we also want to yield focus back to the field and show the accessory view.